tokio/src/task/blocking.rs

```diff
@@ -39,10 +39,9 @@ cfg_rt_threaded! {
 cfg_blocking! {
     /// Runs the provided closure on a thread where blocking is acceptable.
     ///
-    /// In general, issuing a blocking call or performing a lot of compute in a future without
-    /// yielding is not okay, as it may prevent the executor from driving other futures forward.
-    /// A closure that is run through this method will instead be run on a dedicated thread pool for
-    /// such blocking tasks without holding up the main futures executor.
+    /// Spawns a blocking thread on the currently provided executor.
+    /// More information about the blocking threads is available at 
+    /// [CPU-bound tasks and blocking code](../index.html#cpu-bound-tasks-and-blocking-code)
```

Comment:
It does not necessarily spawn a thread. The threads are reused. Additionally, can you split the link and url with this syntax:
```markdown
at [CPU-bound tasks and blocking code][blocking].

[blocking]: ../index.html#cpu-bound-tasks-and-blocking-code
```
You're also missing a period at the end of the sentence.

Reply:
Sure, I can split the URL. Also, is the following correct then - 
*Runs the provided function on the executor reserved for blocking operations*

Reply:
Well, it's part of the same executor as the async code.. I quite like the phrasing in the sentence above: "It can be thought of as an efficient Iterator for collections of bytes." Perhaps you can merge the two lines together?

Reply:
Okay. So, this is the final comment - 
```
Runs the provided function on the executor reserved for blocking operations. 
It can be thought of as an efficient iterator for collections of bytes. 
More information about the blocking threads is available at [CPU-bound tasks and blocking code][blocking].
[blocking]: ../index.html#cpu-bound-tasks-and-blocking-code
```
Is this okay @Darksonn ?

Reply:
Did you miss up your clipboard?

Reply:
Yes, I did not run `rustfmt` before sending it. I will correct this and send it. Is the above comment okay though?

Reply:
I was referring to the "It can be thought of as an efficient iterator for collections of bytes." part.

Reply:
Oh okay! 
I meant this 
```
Runs the provided function on the executor reserved for blocking operations. 
More information about the blocking threads is available at [CPU-bound tasks and blocking code][blocking].
[blocking]: ../index.html#cpu-bound-tasks-and-blocking-code
```

Reply:
Sure, looks good. Perhaps add an empty line between each pair of lines.

Reply:
I will send the PR right away. Thanks !